Makefile

```diff
@@ -11,6 +14,12 @@ serve:
 
 todo:
 	tools/mdbuild.py --todo
+
+.venv: .venv/bin/python
+.venv/bin/python:
+	virtualenv -p python3 .venv
+	$(PIP) install html2text Markdown mdx-anchors-away mdx-callouts mdx-foldouts linkchecker
+
 sysdeps:
```

Comment:
No, I did this to see if I could build on a mac and spell and such aren't there. I feel like it's not a complete replacement and merely an "alternative" setup at this point.